docs/Other SDL Features/Batch Sending RPCs/index.md

```diff
@@ -27,12 +33,54 @@ sdlManager.send([artwork1, artwork2], progressHandler: { (request, response, err
     print("All requests sent \(success ? "successfully" : "unsuccessfully")")
 }
 ```
+!@
+
+@![android,javaSE,javaEE]
+```java
+List<RPCRequest> rpcs = new ArrayList<>();
+
+// rpc 1
+SubscribeButton subscribeButtonRequestLeft = new SubscribeButton();
+subscribeButtonRequestLeft.setButtonName(ButtonName.SEEKLEFT);
+rpcs.add(subscribeButtonRequestLeft);
+
+// rpc 2
+SubscribeButton subscribeButtonRequestRight = new SubscribeButton();
+subscribeButtonRequestRight.setButtonName(ButtonName.SEEKRIGHT);
+rpcs.add(subscribeButtonRequestRight);
+
+
+sdlManager.sendRPCs(rpcs, new OnMultipleRequestListener() {
+    @Override
+    public void onUpdate(int remainingRequests) {
+
+    }
+
+    @Override
+    public void onFinished() {
+
+    }
+
+    @Override
+    public void onResponse(int correlationId, RPCResponse response) {
+
+    }
+
+    @Override
+    public void onError(int correlationId, RPCResponse response) {
+
+    }
+    });
+
+```
+!@
 
 ## Sending Sequential Requests
 Requests sent sequentially are sent in a set order. The next request is only sent when a response has been received for the previously sent request.
 
-The code example below shows how to create a perform interaction choice set. When creating a perform interaction choice set, the `SDLPerformInteraction` RPC can only be sent after the `SDLCreateInteractionChoiceSet` RPC has been registered by Core, which is why the requests must be sent sequentially.
+The code example below shows how to create a perform interaction choice set. When creating a perform interaction choice set, the @![iOS]`SDLPerformInteraction`!@ @![android,javaSE,javaEE]`performInteraction`!@ RPC can only be sent after the @![iOS]`SDLCreateInteractionChoiceSet`!@ @![android,javaSE,javaEE]`createInteractionChoiceSet`!@ RPC has been registered by Core, which is why the requests must be sent sequentially.
```

Comment:
Android `performInteraction` should be `PerformInteraction`, `createInteractionChoiceSet` should be `CreateInteractionChoiceSet` I believe.

Reply:
I just took that directly from the old android docs. Should I still change it?

Reply:
Yeah

Reply:
fixed